src/index.js

```diff
@@ -1,16 +1,34 @@
 import colors from 'colors/safe';
+import _forEach from 'lodash/collection/forEach';
 
 const IS_ERROR_RE = /^error/;
 
 export default function logEvents(cache, options = {}) {
-  cache.onAny(function (...args) {
-    const out = IS_ERROR_RE.test(this.event) ? 'stderr' : 'stdout';
-    const timestamp = (new Date()).toISOString();
-    const log = [colors.grey(timestamp)];
+  if (options.events) {
+    const events = options.events;
+    _forEach(events, function (event) {
```

Comment:
use `events.forEach(function (event) { ... })` instead; and you won't need `lodash`